src/runway/path.py

```diff
@@ -0,0 +1,273 @@
+"""Runway configuration 'path' settings."""
+# pylint: disable=unused-import
+from typing import Dict, List, Optional, Tuple, Union  # noqa: F401
+
+import os
+import logging
+import six
+
+from .sources.git import Git
+
+LOGGER = logging.getLogger('runway')
+
+
+class Path(object):  # pylint: disable=too-many-instance-attributes
+    """Runway configuration ``path`` settings object.
+
+    Path is responsible for parsing the ``path`` property of a Runway
+    configuration. It then can determine if the path specified is
+    locally sourced or remotely sourced through a service such
+    as `Git`_ or S3.
+
+    Local ``path`` variables are defined relative to the root project folder.
+    The value for this cannot be higher than the Runway config file, it must
+    be at the runway file itself or in a sub directory.
+
+    Example:
+        .. code-block:: yaml
+
+            deployments:
+            - modules:
+                - path: my/local/module.cfn
+                - my/local/module.cfn # same as above
+                - ./ # module is in the root
+
+    When the ``path`` is remote, Runway is responsible for fetching
+    the resource and returning the location of it's cached path.
+    The information for retrieving those sources can be controlled via
+    runway rather than manually retrieving each one.
+
+    Example:
+        .. code-block:: yaml
+
+            deployments:
+            - modules:
+                - path: git::git://github.com/your_handle/your_repo.git//my-module.cfn
+
+
+    The ``path`` structure is based on the
+    encoding found in
+    `Terraform modules <https://www.terraform.io/docs/modules/sources.html>`_.
+
+    The values parsed from the string are as follows:
+
+    **source**: Determine if the source is local or remote. The initial
+    prefix is used to determine this separated by `::` in the string.
+    A path is considered local if it contains no source type value.
+
+    Example:
```

Comment:
If you make this a rubric, it will stand out more. or, does it stand out too much like that?

```
.. rubric:: Example
.. code-block:: yaml
```

Reply:
It actually appears to already format that way without specifying the rubric. 
![image](https://user-images.githubusercontent.com/1512028/71823638-60a8fa00-305d-11ea-86c1-0ed509688357.png)